README.md

```diff
@@ -16,7 +16,9 @@ Currently runs single-threaded, supports only `Gomoku` rules, future plans:
 - MCTS with parallelization
 - Self-learning
 
-A live demo is hosted on: https://apps.yunzhu.li/gomoku
+A live demo is hosted on:
```

Comment:
Can we preserve the original link as well? I will remove it if I stop hosting it myself in the future.